packages-proprietary/tokens/src/components/ams/date-input.tokens.json

```diff
@@ -19,7 +19,7 @@
       },
       "disabled": {
         "background-color": { "value": "{ams.color.background}" },
-        "box-shadow": { "value": "inset 0 0 0 {ams.border.width.s} {ams.color.interactive.disabled}" },
+        "border": { "value": "{ams.border.width.s} solid {ams.color.interactive.disabled}" },
```

Comment:
If we remove the color, we can get rid of this token I think

Reply:
Done.